app/reflexes/products_reflex.rb

```diff
@@ -60,4 +64,11 @@ def product_query_includes
       # ]
     ]
   end
+
+  def current_url
+    url = URI(request.original_url)
+    url.query = url.query.present? ? "#{url.query}&" : ""
+    url.query += "page=#{@page}&per_page=#{@per_page}"
```

Comment:
> I think this would continue to add the same parameters to the url each time you view the next page.

Nop, because `request.original_url` is `http://localhost:3000/admin/products_v3` even is has params.

Reply:
Oh! 👍